src/components/CreateNewNote.jsx

```diff
@@ -0,0 +1,103 @@
+                                >
+                                    <option >High</option>
+                                    <option>Meduim</option>
+                                    <option>Low</option>
+                                    
+                                </select>
+                            </div>
+                            <div>
+                            Tag
+                                <select className="form-text text-align-center"
+                                onChange={(e)=>setTag(e.target.value) } value={tag}
+                                >
+                                    <option>Work</option>
+                                    <option>School</option>
+                                    <option>Excercise</option>
+                                    <option>Home</option>
+                                    <option>Task</option>
+                                    
+                                </select>
+                            </div>
+                            <div>
+                            Color
+                                <select className="form-text text-align-center"
+                                onChange={(e)=>setColor(e.target.value) } value={color}
+                                >
+                                    <option>Red</option>
+                                    <option>Blue</option>
+                                    <option>Yellow</option>
+                                    <option>Green</option>
+                                    <option>Black</option>
+                                    
+                                </select>
+                            </div>
+                        </div>    </div>
+                        <div className="d-flex justify-content-end ">
+                            <div>
+                                <button className="btn btn-success "
+                                    onClick={()=>{
+                                        AddNote(title,content,tag,color,priority)
+                                        setColor("Red")
+                                        setContent("")
+                                        setPriority("High")
+                                        setTag("Work")
+                                        setTitle("")
+                                    }}
+                                >
+                                    Add Note</button>
+                            </div>
+                            <div>
+                                <button className="btn btn-danger "
+                                onClick={()=>{
+                                    setColor("Red")
+                                    setContent("")
+                                    setPriority("High")
+                                    setTag("Work")
+                                    setTitle("")
+                                }}
```

Comment:
If time permits you can refactor this code in function like disableHandler or something more better. This would make code look more cleaner